.github/workflows/pyo3.yml

```diff
@@ -52,12 +53,22 @@ jobs:
             if [ -f /usr/bin/yum ]; then
               # quay.io/pypa/manylinux2014_x86_64:latest
               /usr/bin/yum -y upgrade
-              /usr/bin/yum -y install perl-IPC-Cmd perl-List-MoreUtils openssl-devel python3-pip
+              /usr/bin/yum -y install perl-IPC-Cmd perl-List-MoreUtils openssl-devel python3-pip unzip
               /usr/bin/ln -s /usr/bin/pip3 /usr/bin/pip
             elif [ -f /usr/bin/apt ]; then
               # ghcr.io/rust-cross/manylinux2014-cross:aarch64
               /usr/bin/apt update
-              /usr/bin/apt install -y pkg-config libssl-dev
+              /usr/bin/apt install -y pkg-config libssl-dev unzip
+            fi
+
+            if [ "$(uname -m)" == "aarch64" ]; then
+              curl -L https://github.com/protocolbuffers/protobuf/releases/download/v29.1/protoc-29.1-linux-aarch_64.zip -o protoc.zip
+              unzip protoc.zip
+              export PROTOC=$(pwd)/bin/protoc
+            else
+              curl -L https://github.com/protocolbuffers/protobuf/releases/download/v29.1/protoc-29.1-linux-x86_64.zip -o protoc.zip
+              unzip protoc.zip
+              export PROTOC=$(pwd)/bin/protoc
```

Comment:
The package registries were giving me issues. This ended up being a more reliable way to install and pin within the VMs